### PR TITLE
alert counter to be initialized even the alert count is zero

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
tendrl-bugid: Tendrl/gluster-integration#447

Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@r0h4n @shtripat please review

**Comment by GowthamShanmugam** [comment]
Verified

File: tendrl/gluster_integration/objects/volume_alert_counters/__init__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    ):
+        super(VolumeAlertCounters, self).__init__(*args, **kwargs)
+        self.warning_count = warn_count
+        self.integration_id = integration_id
+        self.volume_id = volume_id
+        self.value = '/clusters/{0}/Volumes/{1}/alert_counters'
+
+    def render(self):
+        self.value = self.value.format(
+            self.integration_id, self.volume_id)
```
Comment (r0h4n) [inline tendrl/gluster_integration/objects/volume_alert_counters/__init__.py]: self.integration_id or NS.tendrl_context.integration_id

File: tendrl/gluster_integration/objects/volume_alert_counters/__init__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+from tendrl.commons import objects
+
+
+class VolumeAlertCounters(objects.BaseObject):
+    def __init__(
+        self,
+        warn_count=0,
+        integration_id='',
```
Comment (r0h4n) [inline tendrl/gluster_integration/objects/volume_alert_counters/__init__.py]: integration_id=None, volume_id=None

File: tendrl/gluster_integration/objects/volume_alert_counters/__init__.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        **kwargs
+    ):
+        super(VolumeAlertCounters, self).__init__(*args, **kwargs)
+        self.warning_count = warn_count
+        self.integration_id = integration_id
+        self.volume_id = volume_id
+        self.value = '/clusters/{0}/Volumes/{1}/alert_counters'
+
+    def render(self):
+        self.value = self.value.format(
```
Comment (shtripat) [inline tendrl/gluster_integration/objects/volume_alert_counters/__init__.py]: self.vale.format(self.integration_id or NS.tendrl_context.integration_id, self.volume_id)

File: tendrl/gluster_integration/sds_sync/__init__.py [hunk 4]
```diff
@@ -250,7 +252,16 @@ def run(self):
                     _cluster.last_sync = str(tendrl_now())
                     _cluster.is_managed = "yes"
                     _cluster.save()
-
+                    # Initialize alert count
+                    try:
+                        clus_key = '/clusters/%s/alert_counters' % (
```
Comment (shtripat) [inline tendrl/gluster_integration/sds_sync/__init__.py]: s/clus_key/alerts_count_key/

File: tendrl/gluster_integration/sds_sync/__init__.py [hunk 5]
```diff
@@ -393,6 +404,19 @@ def sync_volumes(volumes, index, vol_options):
         )
         volume.save(ttl=SYNC_TTL)
 
+        # Initialize volume alert count
+        try:
+            vol_key = '/clusters/%s/Volumes/%s/alert_counters' % (
```
Comment (shtripat) [inline tendrl/gluster_integration/sds_sync/__init__.py]: s/vol_key/volume_alert_count_key/

File: tendrl/gluster_integration/objects/volume_alert_counters/__init__.py (hunk 3 above)
Comment (GowthamShanmugam) [reply tendrl/gluster_integration/objects/volume_alert_counters/__init__.py]: ok i will change it

File: tendrl/gluster_integration/objects/volume_alert_counters/__init__.py (hunk 2 above)
Comment (GowthamShanmugam) [reply tendrl/gluster_integration/objects/volume_alert_counters/__init__.py]: i will change it

File: tendrl/gluster_integration/objects/volume_alert_counters/__init__.py (hunk 1 above)
Comment (GowthamShanmugam) [reply tendrl/gluster_integration/objects/volume_alert_counters/__init__.py]: i will change this

File: tendrl/gluster_integration/sds_sync/__init__.py (hunk 4 above)
Comment (GowthamShanmugam) [reply tendrl/gluster_integration/sds_sync/__init__.py]: i will changed like this

File: tendrl/gluster_integration/sds_sync/__init__.py (hunk 5 above)
Comment (GowthamShanmugam) [reply tendrl/gluster_integration/sds_sync/__init__.py]: i will change this

**Comment by GowthamShanmugam** [comment]
@r0h4n @shtripat changes done

**Pull request merged**